ui/src/localStorage.js

```diff
@@ -2,9 +2,22 @@ export const loadLocalStorage = () => {
   try {
     const serializedState = localStorage.getItem('state')
 
-    return JSON.parse(serializedState) || {}
+    const state = JSON.parse(serializedState)
+
+    // eslint-disable-next-line no-undef
+    if (state.VERSION !== VERSION) {
+      // eslint-disable-next-line no-console
+      console.log('New version detected. Clearing old settings.')
+      window.localStorage.removeItem('state')
+      return {}
+    }
+
+    delete state.VERSION
```

Comment:
What are your thoughts on handling this via Redux? That would give us an added benefit of being able to diagnose the chain of events for user-reported errors, if we go with the Redux dump going forward.